Integrations/integration-ReversingLabsTitaniumCloud.yml

```diff
@@ -0,0 +1,221 @@
+        contents = requests.get(base_url + endpoint.format(hash_value=hash_value, hash_type=hash_type),auth=auth).json()['rl']['malware_presence']
+
+        md = ''
+        md += '***\n'
+        md += '## ReversingLabs Malware Presence:\n'
+        md += 'Malware Status: **{mwp_status}**\n'.format(mwp_status=contents['status'])
+
+        try:
+            md += 'First Seen: **' + contents['first_seen'] + '**\n'
+            md += 'Last Seen: **' + contents['last_seen'] + '**\n'
+            md += 'Positives / Total: **' + str(contents['scanner_match']) + '/' + str(contents['scanner_count']) + '**\n'
+        except:
+            None
+
+        ec = {}
+        addFile = {}
+        ec['DBotScore'] = []
+        addFile[outputPaths['file']] = {
+            'MD5': md5,
+            'SHA1': sha1,
+            'SHA256': sha256,
+        }
+
+        dbot_score_map = {'UNKNOWN':0,'KNOWN': 1, 'SUSPICIOUS': 2, 'MALICIOUS': 3}
+
+        if contents['status'] == 'UNKNOWN':
+            addFile[outputPaths['file']]['Unknown'] = {'Vendor': 'ReversingLabs', 'Detections': contents['scanner_match'], 'TotalEngines': contents['scanner_count']}
+            addFile[outputPaths['file']]['properties_to_append'] = ['Unknown']
+
+        elif contents['status'] == 'KNOWN':
+            addFile[outputPaths['file']]['Known'] = {'Vendor': 'ReversingLabs', 'Detections': contents['scanner_match'], 'TotalEngines': contents['scanner_count']}
+            addFile[outputPaths['file']]['properties_to_append'] = ['Known']
+
+        elif contents['status'] == 'SUSPICIOUS':
+            addFile[outputPaths['file']]['Suspicious'] = {'Vendor': 'ReversingLabs', 'Detections': contents['scanner_match'], 'TotalEngines': contents['scanner_count']}
+            addFile[outputPaths['file']]['properties_to_append'] = ['Suspicious']
+
+        elif contents['status'] == 'MALICIOUS':
+            addFile[outputPaths['file']]['Malicious'] = {'Vendor': 'ReversingLabs', 'Detections': contents['scanner_match'], 'TotalEngines': contents['scanner_count']}
+            addFile[outputPaths['file']]['properties_to_append'] = ['Malicious']
+
+        ec[outputPaths['file']] = []
+        ec[outputPaths['file']].append(addFile[outputPaths['file']])
+
+        ec['DBotScore'].append({'Indicator': md5, 'Type': 'hash', 'Vendor': 'ReversingLabs', 'Score': dbot_score_map[contents['status']]})
+        ec['DBotScore'].append({'Indicator': sha1, 'Type': 'hash', 'Vendor': 'ReversingLabs', 'Score': dbot_score_map[contents['status']]})
+        ec['DBotScore'].append({'Indicator': sha256, 'Type': 'hash', 'Vendor': 'ReversingLabs', 'Score': dbot_score_map[contents['status']]})
+        return md, ec
+
+
+    if demisto.command() == 'test-module':
+        try:
+            contents = requests.get("https://ticloud-cdn-api.reversinglabs.com/api/databrowser/rldata/query/md5/6a95d3d00267c9fd80bd42122738e726?extended=true&format=json",auth=auth).json()['rl']
```

Comment:
Is `https://ticloud-cdn-api.reversinglabs.com` is hard coded on purpose? Why not to use the `params.base`?

Reply:
Good catch